README.md

```diff
@@ -138,7 +159,8 @@ aggregation from Drupal. Go to /admin/config/development/performance and uncheck
 site caches and you should be able to continue with your work.
 
 ### How can I add custom translations?
-Add your UI translations to ``./translations/{fi/sv}.po`` files like it is explained in Translation in Drupal 8 documentation: https://www.drupal.org/docs/understanding-drupal/translation-in-drupal-8.
+Add your UI translations to ``./translations/{fi/sv}.po`` files like it is explained in Translation in Drupal 8
```

Comment:
> Add your UI translations to ``./translations/{fi/sv}.po`` files like it is explained in Translation in Drupal 8

Suggestion: 
Add your UI translations to `./translations/{fi/sv}.po` files like it is explained in Translation in Drupal

Reply:
Good catch 😄